src/cmds/query.ts

```diff
@@ -76,6 +83,7 @@ async function runQuery(
     body: JSON.stringify({
       query,
       operationName,
+      variables: JSON.parse(variables)
```

Comment:
Can you put a try/catch around the JSON parsing so you can show a proper error message if it fails?